File: 01_Language/01_Functions/js/long2ip.js
function long2ip(properAddress) {
    return (properAddress >>> 24 & 0xFF) +
        '.' + (properAddress >>> 16 & 0xFF) +
        '.' + (properAddress >>> 8 & 0xFF) +
        '.' + (properAddress & 0xFF)
}

console.log(long2ip('3245348745'))

File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/Long2ip.java
package com.onevgo.functions;

public class Long2ip {
    public static String long2ip(long properAddress) {
        return (properAddress >>> 24 & 0xFF) + "."
                + (properAddress >>> 16 & 0xFF) + "."
                + (properAddress >>> 8 & 0xFF) + "."
                + (properAddress & 0xFF);
    }

    public static void main(String[] args) {
        System.out.println(Ip2long.ip2long("193.112.23.137"));
        System.out.println(long2ip(3245348745L));
    }
}
